todolist/src/store.js

```diff
@@ -9,6 +9,9 @@ export default new Vuex.Store({
     id: 0
   },
   getters: {
+    allTasks: function(state) {
```

Comment:
下記のような省略記法もありますので積極的に利用しましょう。
```suggestion
    allTasks(state) {
```